### PR TITLE
Blog post: first Paris XMPP meetup

### PR DESCRIPTION
**Pull request by Nyco**
I'll need reviews, and publication this week preferably Friday 24th.

### CODE REVIEW

**Comment by guusdk** [comment]
blog-1

Title is a bit to long (it wraps). The category is also pretty long, and partially overlaps with other categories.

File: content/posts/blog/first-paris-xmpp-meetup.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+title: First Paris XMPP/Jabber meetup, March 28th
+date: 2017-03-23 23:59
+author: Nÿco
+blog_id: blog
+category: Summits, Conferences, And Meetups
+---
+
+After Berlin, Stockholm, London, here is Paris. The trend of new XMPP meetups in major cities in Europe is confirmed.
+
+The Paris meetup is very simply organised by the JabberFr community, as this will be mostly oriented discussions our the modernity and contemporaneity of XMPP/Jabber.
```
Comment (SamWhited) [inline content/posts/blog/first-paris-xmpp-meetup.md]: > is very simply organised

I'm not sure what "very simply" means in this context.

> modernity and contemporaneity

I'm not sure "modernity" or "contemporaneity" actually makes sense here or mean what you think they mean; also, I am a native English speaker, and I can't pronounce "contemporaneity" on the first try without getting toung tied; might want to reconsider that one :)

File: content/posts/blog/first-paris-xmpp-meetup.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: First Paris XMPP/Jabber meetup, March 28th
+date: 2017-03-23 23:59
+author: Nÿco
+blog_id: blog
+category: Summits, Conferences, And Meetups
+---
+
+After Berlin, Stockholm, London, here is Paris. The trend of new XMPP meetups in major cities in Europe is confirmed.
```
Comment (SamWhited) [inline content/posts/blog/first-paris-xmpp-meetup.md]: "and London", maybe? I think it reads better, but I'm not srue if it's actually necessary.

File: content/posts/blog/first-paris-xmpp-meetup.md [hunk 3]
```diff
@@ -6,8 +6,8 @@ blog_id: blog
 category: Summits, Conferences, And Meetups
 ---
 
-After Berlin, Stockholm, London, here is Paris. The trend of new XMPP meetups in major cities in Europe is confirmed.
+After Berlin, Stockholm, and London, here is Paris. The trend of new XMPP meetups in major cities in Europe is confirmed.
 
-The Paris meetup is very simply organised by the JabberFr community, as this will be mostly oriented discussions our the modernity and contemporaneity of XMPP/Jabber.
+The Paris meetup is simply organised by the JabberFr community, as this will be mostly oriented discussions on the modernity of XMPP/Jabber.
```
Comment (SamWhited) [inline content/posts/blog/first-paris-xmpp-meetup.md]: Sorry, I think I wasn't clear; "simply organised" still doesn't make sense to me. What does it mean to organize something simply?

Also, modernity does not mean "discussing modern XMPP", modernity is a post-medieval historical period.

**Comment by guusdk** [comment]
Could you change the category? You made up a new one that partially overlaps with the pre-existing categories.

File: content/posts/blog/first-paris-xmpp-meetup.md (hunk 3 above)
Comment (Nyco) [inline content/posts/blog/first-paris-xmpp-meetup.md]: `Sorry, I think I wasn't clear; "simply organised" still doesn't make sense to me. What does it mean to organize something simply?
`
Without waste, very few preparation, nothing fancy like a slidedeck, just two subjects and the open discussions that go around.

`Also, modernity does not mean "discussing modern XMPP", modernity is a post-medieval historical period.
`

Oh I see, thanks for explaining. Well at least post-medieval is no medieval anymore, which is an improvement. ;-)

**Comment by Nyco** [comment]
@guusdk what category should I put?

File: content/posts/blog/first-paris-xmpp-meetup.md (hunk 3 above)
Comment (SamWhited) [inline content/posts/blog/first-paris-xmpp-meetup.md]: > Without waste, very few preparation, nothing fancy like a slidedeck, just two subjects and the open discussions that go around.

Yah, that didn't parse for me. I'm not sure that I have a better suggestion; I'd just leave out "simply" personally.

> on the modern

"on modern XMPP/Jabber" (no article necessary)

**Comment by guusdk** [comment]
I would give it the category "Events".

**Comment by guusdk** [comment]
nyco

**Pull request merged**